src/aiomealie/mealie.py

```diff
@@ -75,6 +87,14 @@ async def _request(self, uri: str, params: dict[str, Any] | None = None) -> str:
             msg = "Unauthorized access to Mealie"
             raise MealieAuthenticationError(msg)
 
+        if response.status == 422:
+            text = await response.text()
+            msg = "Mealie validation error"
+            raise MealieError(
```

Comment:
Let's add a new exception so we can easily see if the issue is a validation issue or a connection issue